src/zcl/definition/cluster.ts

```diff
@@ -4230,7 +4230,19 @@ const Cluster: {
                     {name: 'dpValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
                 ],
             },
-
+            /**
+             * FIXME: This command is not listed in Tuya zigbee cluster description,
+             *  but there is some command 0x05 (description is: Status query)
+             *  in `Serial command list` section of the same document
+             *  So, need to investigate more information about it
```

Comment:
What more needs to be investigated here?

Reply:
I just copied the comments from section below (ID 6) and assume it's the same for command 0x06 and 0x05.
To be honest, I did not have the time to look at the [docs](https://developer.tuya.com/en/docs/iot-device-dev/tuya-zigbee-universal-docking-access-standard?id=K9ik6zvofpzql#subtitle-6-Private%20cluster)

Keep the comment in source code or remove it?

Reply:
If it works please remove it.

Also, where is this command used? I would expect a pr to zigbee-herdsman-converters which handles this.

Reply:
I will merge them together.